tests/test_views.py

```diff
@@ -4583,5 +4529,269 @@ def test_confirm_user_signal_called_when_user_confirms_email(self):
         assert_equal(mock_signals.signals_sent(), set([auth.signals.user_confirmed]))
 
 
+class TestDraftRegistrationViews(OsfTestCase):
+
+    def setUp(self, *args, **kwargs):
+        super(TestDraftRegistrationViews, self).setUp(*args, **kwargs)
+
+        self.user = AuthUserFactory()
+        self.auth = self.user.auth
+        self.node = ProjectFactory(creator=self.user)
+
+        MetaSchema.remove()
+        ensure_schemas()
+        self.meta_schema = MetaSchema.find_one(
+            Q('name', 'eq', 'Open-Ended Registration') &
+            Q('schema_version', 'eq', 1)
+        )
+        self.draft = DraftRegistrationFactory(
+            initiator=self.user,
+            branched_from=self.node,
+            registration_schema=self.meta_schema,
+            registration_metadata={
+                'summary': {'value': 'Some airy'}
+            }
+        )
+        self.draft.save()
+
+        current_month = datetime.datetime.now().strftime("%B")
+        current_year = datetime.datetime.now().strftime("%Y")
+
+        valid_date = datetime.datetime.now() + datetime.timedelta(days=180)
+        self.valid_embargo_payload = json.dumps({
```

Comment:
This payload- and the tests that use it- are part of this PR, and you wouldn't be implementing new methods- just using existing functionality in the current test library.

Relatively small. I'll consider making it outside the scope of the PR if these tests are being rewritten in #4280 (the other half of this work)... but otherwise, my preference is to consider code introduced in this PR in-scope, by definition. Always open to further discussion in person if needed.